content/en/monitors/notify/_index.md

```diff
@@ -116,6 +122,25 @@ Disk space is low @ops-team@company.com
 
 {{% notifications-integrations %}}
 
+### Workflows
+You can trigger a [workflow automation][9] or create a new workflow from a monitor.
+
+**To add an existing workflow to a monitor**:
+1. In the message section, add the full workflow mention name:
+   - The mention name should start with `@workflow-`. For example, `@workflow-my-workflow`
+   - To pass trigger variables into the workflow, use a comma-separated list with the syntax `@workflow-name(key=value, key=value)`. You can use message template variables as trigger variables. For example, `@workflow-my-workflow(hostname={{host.name}})`
```

Comment:
@hestonhoffman For this: "For example, `@workflow-my-workflow(hostname={{host.name}})`" we don't use the curly braces: For example, `@workflow-my-workflow(hostname=host.name)`